packages/eas-cli/src/utils/statuspageService.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  StatuspageServiceFragment,
+  StatuspageServiceName,
+  StatuspageServiceStatus,
+} from '../graphql/generated';
+import { StatuspageServiceQuery } from '../graphql/queries/StatuspageServiceQuery';
+import Log from '../log';
+
+export async function warnAboutEasOutagesAsync(
+  serviceNames: StatuspageServiceName[]
+): Promise<void> {
+  const services = await getStatuspageServiceAsync(serviceNames);
+
+  services.forEach(warnAboutServiceOutage);
+}
+
+function warnAboutServiceOutage(service: StatuspageServiceFragment): void {
+  const humanReadableServiceName: Record<StatuspageServiceName, string> = {
+    [StatuspageServiceName.EasBuild]: 'EAS Build',
+    [StatuspageServiceName.EasSubmit]: 'EAS Submit',
+    [StatuspageServiceName.EasUpdate]: 'EAS Update',
+  };
+
+  if (service.status !== StatuspageServiceStatus.Operational) {
+    Log.warn(
+      `⚠️  ${
+        humanReadableServiceName[service.name]
+      } is currently in a degraded state. The service may temporarily not function as expected.`
```

Comment:
```suggestion
      } may temporarily not function as expected.`
```

ideally we'd be able to say what kind of outage is reported - partial or major. so, something like

> "is experiencing a partial outage."

or

> "is experiencing a major outage."

Reply:
That sounds like a good idea, thank you!

Reply:
<img width="863" alt="Screenshot 2022-08-26 at 10 38 47" src="https://user-images.githubusercontent.com/55145344/186861664-03fbb819-f79e-445b-89f8-a94375c2188e.png">
This is how the communicate looks after all of the changes